staging/src/k8s.io/component-base/README.md

```diff
@@ -0,0 +1,34 @@
+## component-base
+
+## Purpose
+
+Implement KEP 32: https://github.com/kubernetes/enhancements/blob/master/keps/sig-cluster-lifecycle/0032-create-a-k8s-io-component-repo.md
```

Comment:
Use `[KEP 32: Create a k8s.io/component-base repo](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cluster-lifecycle/0032-create-a-k8s-io-component-repo.md)` instead?

Reply:
may improve in a later PR